.github/workflows/container-autoupdate.yml

```diff
@@ -0,0 +1,34 @@
+name: Build, test and push container
+on:
+  schedule:
+    - cron: 0 0 * * *
+  workflow_dispatch:
+
+jobs:
+  push-container:
+    name: Push container to Quay.io
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout anaconda repository
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Quay.io
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
```

Comment:
I suggest committing them to our internal secrets repo. Also, any chance to use a less privileged token for the upload instead of the full user account? (not sure if quay supports that)

Reply:
That is a bot account from Quay.io. It has Write access to push the images.

Reply:
OK, fair enough.